README.md

```diff
@@ -8,5 +8,12 @@ The snap consists of [dcgm](https://developer.nvidia.com/dcgm) and [dcgm-exporte
 You can build the snap locally by using the command:
 
 ```shell
-snapcraft --use-lxd
-```
\ No newline at end of file
+make build
+```
+
+## Install the snap
+
+```shell
+snap install --dangerous ./dcgm.snap
+sudo systemctl enable snap.dcgm.dcgm-exporter.service --now # enable and start dcgm-exporter service
```

Comment:
Once the snap is available at the snap store there is no need for a normal user to install it locally.

I don't think it's necessary to have this part at the README. Maybe we can do in the future some instructions to install locally at the `CONTRIBUTING.md`

Reply:
I will leave it as is, I will rework the README.md entirely in the next PR.